memgpt/agent.py

```diff
@@ -1053,25 +1068,41 @@ def remove_function(self, function_name: str) -> str:
     #        self.ms.update_agent(agent=new_agent_state)
 
     def update_state(self) -> AgentState:
-        updated_state = {
+        # updated_state = {
+        #    "persona": self.memory.persona,
+        #    "human": self.memory.human,
+        #    "system": self.system,
+        #    "functions": self.functions,
+        #    "messages": [str(msg.id) for msg in self._messages],
+        # }
+        memory = {
+            "system": self.system,
             "persona": self.memory.persona,
             "human": self.memory.human,
-            "system": self.system,
-            "functions": self.functions,
-            "messages": [str(msg.id) for msg in self._messages],
+            "messages": [str(msg.id) for msg in self._messages],  # TODO: move out into AgentState.message_ids
+        }
+
+        # TODO: add this field
+        metadata = {  # TODO
+            "human_name": self.agent_state.persona,

```

Comment:
is this correct? is the name in the right place, and the value in the right place?